test/powershell/Modules/Microsoft.PowerShell.Management/Clear-Content.Tests.ps1

```diff
@@ -75,35 +77,46 @@ Describe "Clear-Content cmdlet tests" -Tags "CI" {
       $cci.SupportsShouldProcess | Should -BeTrue
     }
 
-    It "Alternate streams should be cleared with clear-content" -Skip:(!$IsWindows) {
-      # make sure that the content is correct
-      # this is here rather than BeforeAll because only windows can write to an alternate stream
-      Set-Content -Path "TestDrive:/$file3" -Stream $streamName -Value $streamContent
-      Get-Content -Path "TestDrive:/$file3" | Should -BeExactly $content2
-      Get-Content -Path "TestDrive:/$file3" -Stream $streamName | Should -BeExactly $streamContent
-      Clear-Content -Path "TestDrive:/$file3" -Stream $streamName
-      Get-Content -Path "TestDrive:/$file3" | Should -BeExactly $content2
-      Get-Content -Path "TestDrive:/$file3" -Stream $streamName | Should -BeNullOrEmpty
-    }
-
-    It "the '-Stream' dynamic parameter is visible to get-command in the filesystem" -Skip:(!$IsWindows) {
-      try {
-        Push-Location -Path TestDrive:
-        (Get-Command Clear-Content -Stream foo).parameters.keys -eq "stream" | Should -Be "stream"
+    Context "Clear-Content should work with alternate data streams on Windows" {
+      It "Alternate streams should be cleared with Clear-Content on a file" -Skip:(!$IsWindows) {
+        # Make sure the test is set up correctly.
+        Set-Content           -Path "TestDrive:/$file3" -Stream $streamName -Value $streamContent
+        Get-Content           -Path "TestDrive:/$file3" -Stream $streamName | Should -BeExactly $streamContent
+        # Truncate the stream.
+        Clear-Content         -Path "TestDrive:/$file3" -Stream $streamName -ErrorAction Stop
+        # The stream should exist, but should have 0 length.
+        $result = Get-Item -Path "TestDrive:/$file3" -Stream $streamName
```

Comment:
No need to add obvious comments. We can separate logical blocks with blank lines.
In next test too.

```suggestion
        Set-Content           -Path "TestDrive:/$file3" -Stream $streamName -Value $streamContent
        Get-Content           -Path "TestDrive:/$file3" -Stream $streamName | Should -BeExactly $streamContent

        Clear-Content         -Path "TestDrive:/$file3" -Stream $streamName -ErrorAction Stop

        $result = Get-Item -Path "TestDrive:/$file3" -Stream $streamName
```

Reply:
Okay, fixing.